src/components/tooltip/tooltip.js

```diff
@@ -104,8 +104,9 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $interpolate,
 
     function addAriaLabel(override) {
```

Comment:
@DevVersion - please add comment (in another PR) that describes what `override` does?  

> One has to read the entire component code to release that it is used as the changeHandler in a `.$watch( )`; so the `override` is the value interpolated from $watch.